pkgs/development/go-modules/generic/default.nix

```diff
@@ -51,6 +51,12 @@
 , buildFlags ? ""
 , buildFlagsArray ? ""
 
+# string or list of strings used to filter out packages
+, excludedPackages ? ""
+
+# string or list of strings used to determine packages to build
+, subPackages ? ""
```

Comment:
Please explain why `excludedPackages` and `subPackages` need to be explicity included here (and in `go-packages`)?

Your commit message says:
> I had to read the implementation to figure out how to use subPackages which
should have been documented better to begin with. This would be better to have in
the manual, but this is at least better than before.

But doesn't give a technical reason for them to be here?

They shouldn't be added just for documentation and you have also added/updated them in the go docs so the reason stated in the commit message is no longer valid?

Reply:
Well its not just for documentation, these are inputs to the function and seems like they should show up explicitly instead of just coming in implicitly via the `...` right?

Reply:
No, please remove them.

Reply:
feels weird to me tbh, but ok.